src/mjolnir/util.cc

```diff
@@ -232,6 +233,7 @@ bool build_tile_set(const boost::property_tree::ptree& original_config,
                     const std::vector<std::string>& input_files,
                     const BuildStage start_stage,
                     const BuildStage end_stage) {
+  SCOPED_TIMER();
```

Comment:
i think its ok to leave this one in, since the function name will be there in the plot it will be obvious that its the "total" time to do the whole build